web/.storybook/.babelrc

```diff
@@ -1,8 +1,3 @@
 {
-  "extends": "./../packages/build/.babelrc",
-  "presets": [
-    "@babel/preset-react",
-    "@babel/preset-typescript"
```

Comment:
Our babelrc for Storybook used to include a subset of presets. This was done to optimize the build time of Storybook by avoiding the use of `preset-env` when running Storybook (https://github.com/gravitational/webapps/pull/473#issuecomment-982024260). AFAIK in the current version of Babel this had no effect – [`extends` merges one config with the other](https://babeljs.io/docs/configuration#how-babel-merges-config-items), so any preset present in the config that we're extending will be also present in the final config.